resources/kcp/charts/provisioner/templates/deployment.yaml

```diff
@@ -118,6 +118,8 @@ spec:
               value: {{ .Values.support.l2OperatorRoleBindingSubject | quote }}
             - name: APP_OPERATOR_ROLE_BINDING_L3SUBJECT_NAME
               value: {{ .Values.support.l3OperatorRoleBindingSubject | quote }}
+            - name: APP_OPERATOR_ROLE_BINDING_CREATING_FOR_ADMIN
```

Comment:
Do we use in the queue creation?
Does it refer to this lines https://github.com/kyma-project/control-plane/pull/722/files#diff-ea94d159b95db4e74bbd041a8f2d3dc1fd9d40862cd719c96d6007485ad5b28fR91-R98 ?

Reply:
In my opinion it's working, and in this line this value is used.

Reply:
Yes, I confirmed it today. Wasn't sure.